### ISSUE TITLE
FEATURE: Add expiration date and Complete in

### ISSUE THREAD

**Issue by martinskalicky** [recursivetree/seat-transport#14]
This wont change any behaviour but for visual purposes might be better.

If we can add expiration date and complete in to the settings of the routes that would be perfect. So people have it in one table below each other and we dont have to write it in the text box which we can have to each route (similar settings as max collateral)

This is low prio TBH its just visual. Might be more clear to people

**Comment by recursivetree** [comment]
This is what I intended the info text to be used for

**Comment by martinskalicky** [comment]
True. as I said its visual right. If u want to compare whats in calculator and whats in the contract window in the game. Just a bit faster checking. Not a bit deal
